### PR TITLE
Copy only the error message

### PR DESCRIPTION
**Pull request by Akarshit**
The message "NetLogo is unable to supply you with more details about this error.  Please report the problem at https://github.com/NetLogo/NetLogo/issues, or to bugs@ccl.northwestern.edu, and paste the contents of this window into your report." does not get copied when clicking the copy button.

### CODE REVIEW

**Comment by mrerrormessage** [comment]
Thanks for the PR, I'll review tomorrow.

**Comment by mrerrormessage** [comment]
Thanks for the PR! It works as described, but it that isn't always desirable.

Sometimes the first part of an error message contains information that's would be really valuable in figuring out the problem. For instance if I type `runresult "null"` into the command center, I see the following error:
> Nothing named NULL has been defined
> error while observer running RUNRESULT
>   called by Command Center
>
> org.nlogo.nvm.EngineException: Nothing named NULL has been defined
> at org.nlogo.prim.etc._runresult.report(_runresult.scala:34)
> ...
Under this PR, the first part of that (which could be helpful in debugging) isn't copied. The best thing to do would be to truncate the first part of the message only if we know the message is: "NetLogo is unable to supply you with more details about this error...". Does that sound alright to you?

**Comment by SethTisue** [comment]
```scala
scala> "foobar".stripPrefix("foo")
res1: String = bar

scala> "bazbar".stripPrefix("foo")
res2: String = bazaar
```

**Comment by Akarshit** [comment]
Thanks @mrerrormessage for pointing out the missing case and @SethTisue for the idea.
I have changed the commit and the concept now used is similar to what @SethTisue mentioned.
Please mention if i am still neglecting any use case.

File: src/main/org/nlogo/window/RuntimeErrorDialog.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
   private String eventTrace;
   private final String className;
   private String threadName;
+  String extra;
```
Comment (mrerrormessage) [inline src/main/org/nlogo/window/RuntimeErrorDialog.java]: please make `private`.

**Comment by mrerrormessage** [comment]
Looks good, just one comment and then we can merge this in.

**Comment by Akarshit** [comment]
@mrerrormessage Done.

**Pull request merged**